packages/frontend/src/utils/tvl/getTvlWitchChange.ts

```diff
@@ -0,0 +1,16 @@
+import { Bridge, Layer2 } from '@l2beat/config'
+import { TvlApiResponse } from '@l2beat/types'
+
+import { getPercentageChange } from '../utils'
+
+export function getTvlWithChange(
```

Comment:
You should probably use this on tvl pages as well, because now we have duplicated functionality